src/comparator.html

```diff
@@ -0,0 +1,252 @@
+        var zoom = master.getZoom();
+        var bearing = master.getBearing();
+        var pitch = master.getPitch();
+
+        clones.forEach(function (clone) {
+          clone.jumpTo({
+            center: center,
+            zoom: zoom,
+            bearing: bearing,
+            pitch: pitch,
+          });
+        });
+      }
+      function syncMaps() {
+        var maps;
+        var argLen = arguments.length;
+        if (argLen === 1) {
+          maps = arguments[0];
+        } else {
+          maps = [];
+          for (var i = 0; i < argLen; i++) {
+            maps.push(arguments[i]);
+          }
+        }
+
+        // Create all the movement functions, because if they're created every time
+        // they wouldn't be the same and couldn't be removed.
+        var fns = [];
+        maps.forEach(function (map, index) {
+          fns[index] = sync.bind(
+            null,
+            map,
+            maps.filter(function (o, i) {
+              return i !== index;
+            })
+          );
+        });
+
+        function on() {
+          maps.forEach(function (map, index) {
+            map.on('move', fns[index]);
+          });
+        }
+
+        function off() {
+          maps.forEach(function (map, index) {
+            map.off('move', fns[index]);
+          });
+        }
+
+        // When one map moves, we turn off the movement listeners
+        // on all the maps, move it, then turn the listeners on again
+        function sync(master, clones) {
+          off();
+          moveToMapPosition(master, clones);
+          on();
+        }
+
+        on();
+        return function () {
+          off();
+          fns = [];
+        };
+      }
+
+      function addLicense(url, resourceType) {
+        var licensedUrl = new URL(url);
+        var params = licensedUrl.searchParams;
+        params.set('license', 'cff9c99a-f789-4e74-890e-a1b41b6cb437');
+        params.set('my_app_name', 'ems-basemap-editor');
+        params.set('my_app_version', '1.0.0');
+        return {
+          url: licensedUrl.toString(),
+        };
+      }
+
+      var atlas = {
+        world: {
+          id: 'world',
+          center: [0, 0],
+          zoom: 1,
+        },
+        london: {
+          id: 'london',
+          center: [-0.14, 51.53],
+          zoom: 6,
+        },
+        dc: {
+          id: 'dc',
+          center: [-77.033, 38.888],
+          zoom: 10,
+        },
+        berlin: {
+          id: 'berlin',
+          center: [13.392, 52.519],
+          zoom: 12,
+        },
+        sydney: {
+          id: 'sydney',
+          center: [-208.793, -33.87],
+          zoom: 14,
+        },
+      };
+
+      var userMaps = [];
+      let style = 'osm-bright';
+
+      if ('URLSearchParams' in window) {
+        // Browser supports URLSearchParams
+        var params = new URLSearchParams(window.location.search);
+        if (params.has('style')) {
+          style = params.get('style');
+        }
```

Comment:
```suggestion
        } else {
          var dialog = document.querySelector('dialog');
          dialog.classList.remove('dn');
          dialogPolyfill.registerDialog(dialog);
          dialog.showModal();
        }
```

If no `style` param is present, remove the `dn` class and show the modal.